src/client/gameui.cpp

```diff
@@ -85,7 +86,8 @@ void GameUI::init()
 }
 
 void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_control,
-	const CameraOrientation &cam, const PointedThing &pointed_old, float dtime)
+		const CameraOrientation &cam, const PointedThing &pointed_old,
+		const GUIChatConsole *chat_console, float dtime)
```

Comment:
No need for double indent here.

Reply:
Aren't continuation lines supposed to use double-indents?

Reply:
https://dev.minetest.net/Code_style_guidelines#Function_declarations
>Indention for follow up lines is **exactly** one tab.

Reply:
Wow, didn't know we had separate indentation guidelines for function declarations, thanks. :)